apps/cli/templates/defaults/remix.config.js

```diff
@@ -0,0 +1,28 @@
+/** @type {import('@remix-run/dev').AppConfig} */
+module.exports = {
```

Comment:
I think this config is designed for the builder and you don't need in the cli a lot of these things

Reply:
These are all config files. Which are bunched into the `template.json` file. These are again generated on the fly into the end build to create a project. Since the cli runs `npm install` `npm build` to generate the builds. These files need to sit along with the users projects.

Reply:
sure but I don't know why you need all that stuff inside of it, I think you copied it from the builder, but cli for published sites doesn't have these dependences and requirements